src/Components/TMS/Organisms/FOTMSSidebar/index.tsx

```diff
@@ -0,0 +1,248 @@
+import React, { useEffect, memo } from "react";
+// import Router from "next/router";
+import IDashboardSidebarProps from "./FOTMSSidebar.interfaces";
+
+import { Box, Icon, Collapse } from "@mui/material";
+
+import { styled, useTheme } from "@mui/material/styles";
+import Drawer from "@mui/material/Drawer";
+
+import List from "@mui/material/List";
+import Divider from "@mui/material/Divider";
+import IconButton from "@mui/material/IconButton";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
+import ChevronLeftIcon from "@mui/icons-material/ChevronLeft";
+import ListItem from "@mui/material/ListItem";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+
+import Button from "@mui/material/Button";
+import SettingsIcon from "@mui/icons-material/Settings";
+import ExpandLess from "@mui/icons-material/ExpandLess";
+import ExpandMore from "@mui/icons-material/ExpandMore";
+
+const drawerWidth = 300;
+
+const DrawerHeader = styled("div")(({ theme }) => ({
+  display: "flex",
+  alignItems: "center",
+  padding: theme.spacing(0, 1),
+  // necessary for content to be below app bar
+  ...theme.mixins.toolbar,
+  justifyContent: "flex-end",
+}));
+
+export const FOTMSSidebar = memo((props: IDashboardSidebarProps) => {
+  const { open, content } = props;
+  const initialPath = window.location.pathname;
+  const pathMarkerSidebar = (path: string) => {
+    if (initialPath === path) {
+      return {
+        bgcolor: "#FDEBEA",
+        width: "100%",
+        display: "flex",
+        alignItems: "center",
+        borderRadius: 16,
+        padding: "0 12px",
+      };
+    }
+    return {
+      width: "100%",
+      display: "flex",
+      alignItems: "center",
+      borderRadius: 16,
+      padding: "0 12px",
+    };
+  };
+
+  const theme = useTheme();
+  const [drawerOpen, setDrawerOpen] = React.useState(false);
```

Comment:
for boolean value, please add prefi is -> isDrawerOpen

Reply:
noted